src/main/java/org/jitsi/videobridge/shim/VideobridgeShim.java

```diff
@@ -135,6 +136,9 @@
                     = new ColibriConferenceIQ.Channel();
             channelShim.describe(responseChannelIQ);
             createdOrUpdatedChannels.add(responseChannelIQ);
+
+            // TODO: do we still want to support the legacy mode of putting
+            // <transport> inside <channel>?
```

Comment:
this implies non-rtcp mux, doesn't it?  so if anything we should error on that maybe?  otherwise we'd have to do a lot of error checking to make sure the multiple transport descriptions matched

Reply:
It does not imply non-rtcpmux necessarily. For a while at least jicofo used to include the transport in the video channel (but not in the audio channel) and the current bridge code just routes that to the channel's channle-bundle. I'm just wondering if other colibri implementations still do something like this. I'm fine not doing it, though. Maybe I will just log a warning to make it easy to diagnose.